### ISSUE TITLE
A11y_AzureToolsStorageExplorer_AttachToLocalEmulator_ScreenReader: 'Display name:' field is not announced by Voice Over.

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2313]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;  #VoiceOver; #Mac; #CELAAssessID_CosmosDB-FY20-Q2; 

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.0
MAC Version: Catalina 10.15 

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect Dialog" pane present on left side of screen and hit enter.
3. Select "Attach to local emulator" radio button and hit enter on Next button.
4. "Attach to Local Emulator" screen gets opened.
5. Start Voice Over and navigate to display name field.
6. Observe the Voice Over announcement.

**Actual:**
Label of fields - "Display name", "Blobs port", "Files port" and "Queues port" in 'Attach to Local Emulator' dialog are not announced. VoiceOver announces the respective values (if any) in edit boxes on the dialog and no information about the labels.
For eg: for "Display name" field, (having "local-1" as value),  VoiceOver says, "Local-1, Selected".
Note: VoiceOver announces the label of "Tables port" field.

**Same issue is observed in below Screens:**
1. This issue also affects the text field - "Display name:'in 'Connect with Name and Key' dialog.
2.This issue also affects the text field - "Display name:' in 'Attach with SAS URI' dialog.
3. This issue also affects the text field - "Tenant:" in "Attach with Azure AD" dialog.

**Expected:**
When VoiceOver users navigate to the 'Attach to Local Emulator' dialog,  the label of all fields should be announced.
For eg: "Display name, Edit, Selected, Local-1"

**UserImpact:**
Screen reader users may not be able to know the name of the edit field on which they are navigating.

**Comment by MRayermannMSFT** [comment]
Same root cause as https://github.com/microsoft/AzureStorageExplorer/issues/1891 which I have already checked in a fix for.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT 
Bug#[2313](https://github.com/microsoft/AzureStorageExplorer/issues/2313) and Bug#[1891](https://github.com/microsoft/AzureStorageExplorer/issues/1891)are logged for two different Platforms.
Issue#2313 is logged for VoiceOver and Issue#1891 is logged for NVDA/JAWS.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0 in MAC Version: Catalina 10.15, this issue is still reproduced.

**Comment by MRayermannMSFT** [comment]
Unfortunately this appears to be a bug in Chromium. I have two bugs open against Chromium. One is for text inputs (like Display Name) and the other is for number inputs (like Blob Port):

1. https://bugs.chromium.org/p/chromium/issues/detail?id=1024011
2. https://bugs.chromium.org/p/chromium/issues/detail?id=1048735

**Comment by MRayermannMSFT** [comment]
Chromium has accepted the bug. Will go over this with accessibility team on monday.

**Comment by Alexandria-R** [comment]
This was approved to close as low impact.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
